slave-dotnet20/Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM openshift3/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet20-rhel7-docker" \
```

Comment:
This slave specifically targets the 2.0 version of .NET Core. This matches with the scl packaging we have in mind. It means we will have a builder for each version of .NET Core (which are roughly released at 6M intervals).